Engine/IStringConvertProvider.cs

```diff
@@ -832,7 +832,7 @@ public object FromString(string stringdata, ITypeData type, object contextObject
                 if (!type.DescendsTo(typeof(IInput))) return null;
                 
                 var s = stringdata.Split(':');
-                var step = contextObject as ITestStep;
+                var step = contextObject as ITestStep ?? (contextObject as IInput)?.Step;
```

Comment:
```suggestion
                // under normal circumstances the contextObject is a test step
                // but during cloning it may also be another Input<T> base instance.
                var step = contextObject as ITestStep ?? (contextObject as IInput)?.Step;
```

Reply:
Read my other suggestion - maybe we don't need this?

Reply:
I'm a bit on the fence with this. 

It would of course be more sane if the context was always a teststep, but it would also make sense if `InputStringConvertProvider` understood an IInput as context.